src/tickit/adapters/io/tcp_io.py

```diff
@@ -20,9 +20,10 @@ class TcpIo(AdapterIo[CommandAdapter]):
     host: str
     port: int
 
-    def __init__(self, host: str, port: int) -> None:
+    def __init__(self, host: str, port: int, separator: str = None) -> None:
```

Comment:
Should: This shouldn't pass a type check and I'm surprised it does, although it might just be because the CI was disabled.

```suggestion
    def __init__(self, host: str, port: int, separator: str | None = None) -> None:
```